src/lib.rs

```diff
@@ -169,10 +221,10 @@ impl ApplicationBuilder {
         
         let general = config.general.unwrap_or_default();
 
-        let init_path = get_or(&general, "init", "init.lua");
+        let init_path = init_path.unwrap_or(Path::new(&get_or(&general, "init", "init.lua")).to_path_buf());
```

Comment:
You should use `PathBuf::from` instead which would be simple.

Reply:
Nice point, thanks!